### DISCUSSION TITLE
Is it expected that Greasemonkey scripts don't work if JS is disabled?

### DISCUSSION THREAD

**Discussion by triallax** [qutebrowser/qutebrowser#6723] [Q&A] [answered]
If so, are there any workarounds?

**Comment by The-Compiler** [comment, marked as answer]
Yes, they run in the same context as the page. You can add `@qute-js-world jseval` to run it in the same JS context as `:jseval`, but not all scripts will work that way.

**Reply by triallax** [reply]
Thanks, this seems to work for the two simple scripts I have (they just insect some CSS into the page). Maybe it would be wise to document this somewhere?
